samples/SpriteFrames/CMakeLists.txt

```diff
@@ -1,36 +1,15 @@
 cmake_minimum_required(VERSION 3.0)
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/bin/linux)
-set(CMAKE_BUILD_TYPE Debug)
-
 project(SpriteFramesSample)
 
-add_definitions(-std=gnu++11)
```

Comment:
because i've put `set (CMAKE_CXX_STANDARD 11)` at the root parent cmakelists.txt, this is no longer required.